README.md

```diff
@@ -300,6 +286,16 @@ In config page template
 {{/tableSections}}
 ```
 
+This assumes all steps containing fields have a `section` - `locals.section`, this is used to group fields in the confirm table and the email.
```

Comment:
It's a good point though - it should be able to handle lack of sections, we introduced that so probably not relevant for all apps. I'll add a code example for now, but we should change this so it supports apps without sections